acceptance/managers/image_manager.go

```diff
@@ -62,6 +62,13 @@ func (im ImageManager) HostOS() string {
 	return daemonInfo.OSType
 }
 
+func (im ImageManager) HostArch() string {
+	im.testObject.Helper()
+	daemonInfo, err := im.dockerCli.Info(context.Background())
```

Comment:
I think I have had some issues with this in the past. Shouldn't be `dockerCli.ServerVersion` ?

Reply:
Updated in f8a10339 (I realized we don't need it)

Reply:
For posterity, the version API is safer because:

```
/info os is: linux
/info arch is: x86_64
/version os is: linux
/version arch is: amd64
```

Version arch is what we want when pulling images